extensions/control-plane/callback/callback-http-dispatcher/src/main/java/org/eclipse/edc/connector/callback/dispatcher/http/CallbackEventDispatcherHttpExtension.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.connector.callback.dispatcher.http;
+
+import org.eclipse.edc.runtime.metamodel.annotation.Extension;
+import org.eclipse.edc.runtime.metamodel.annotation.Inject;
+import org.eclipse.edc.spi.http.EdcHttpClient;
+import org.eclipse.edc.spi.message.RemoteMessageDispatcherRegistry;
+import org.eclipse.edc.spi.system.ServiceExtension;
+import org.eclipse.edc.spi.system.ServiceExtensionContext;
+import org.eclipse.edc.spi.types.TypeManager;
+
+@Extension(value = CallbackEventDispatcherHttpExtension.NAME)
+public class CallbackEventDispatcherHttpExtension implements ServiceExtension {
+    
+    public static final String NAME = "Callback dispatcher http extension";
+    @Inject
+    RemoteMessageDispatcherRegistry registry;
+    @Inject
+    EdcHttpClient client;
+    @Inject
+    TypeManager typeManager;
+
+    @Override
+    public String name() {
+        return NAME;
+    }
+
+    @Override
+    public void initialize(ServiceExtensionContext context) {
+
+        var baseDispatcher = new GenericHttpRemoteDispatcherImpl(client);
+        baseDispatcher.registerDelegate(new CallbackEventRemoteMessageDispatcher(typeManager.getMapper()));
+
+        registry.register(new GenericHttpRemoteDispatcherWrapper(baseDispatcher, "http"));
+        registry.register(new GenericHttpRemoteDispatcherWrapper(baseDispatcher, "https"));
```

Comment:
I would avoid to have http/https specific dispatcher registered, in fact the "protocol" thing speaks the domain language instead of the actual low-level protocol. 
As the current ids protocol is described by `ids-multipart`, and for the upcoming dsp protocol is `dataspace-protocol-http`, I would suggest to define a `callback-event-http` protocol string that will work for both http and https, removing the wrapper thing.

Reply:
I'm not sure we need that, in the same way we don't need such a distinguish in the `ids-multipart` or in the `dataspace-protocol-http`. they will be dispatched in the same way, no differences between `http` and `https`

Reply:
yeah but in this case what drives the protocol is the scheme part of the URL. If we want to change the protocol name to `callback-event-http` probably we would need a mapping between the schema part and `callback-event-http` so in the future we could have another dispatcher different from http/https